src/src/AdminBundle/Resources/views/EventType/edit.html.twig

```diff
@@ -11,8 +11,8 @@
         </div>
     </div>
     <div class="row">
-        <div class="col-md-6">
-            <a class='btn btn-primary' href="{{ path('event_add') }}">Add new Event</a>
+        <div class="col-md-7">
+            <a class='btn btn-primary' href="{{ path('event_add', {'id':form.vars.value.id}) }}">Add new Event</a>
```

Comment:
`form.vars.value.id` osobne sa mi to nepaci, netusim co to na posadi urobi, nebolo by krajsie si to `render` poslat dalsiu premennu `event_type_id` a pouzit ju vsade kde treba?